packages/matchbox/src/components/ScreenReaderOnly/ScreenReaderOnly.js

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styles from './ScreenReaderOnly.module.scss';
+import styled from 'styled-components';
+import { visuallyHidden } from '../../styles/helpers';
+
+const StyledScreenReaderOnly = styled('span')`
```

Comment:
How does this work when we used block element inside it?

https://github.com/SparkPost/2web2ui/blob/master/src/components/hibana/DesktopNavigation.js#L29-L31

Reply:
You are definitely right - we should expose an `as` prop. I'll go ahead and do that as well so then I can incorporate it as part of the component wrapper ticket as well.